Tools/clinic/clinic.py

```diff
@@ -575,7 +575,7 @@ def strip_leading_and_trailing_blank_lines(s):
         del lines[0]
     return '\n'.join(lines)
 
-@functools.lru_cache()
```

Comment:
Sorry 'bout the nitpicking, but I think I'd like to leave this change out of the PR; there's no readability improvement with this change.

Reply:
Sure